sdk/storagesync/Azure.ResourceManager.StorageSync/tests/Azure.ResourceManager.StorageSync.Tests.csproj

```diff
@@ -2,4 +2,24 @@
   <ItemGroup>
     <ProjectReference Include="..\src\Azure.ResourceManager.StorageSync.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Properties\Settings.Designer.cs">
+      <DesignTimeSharedInput>True</DesignTimeSharedInput>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>Settings.settings</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="Properties\Settings.settings">
+      <Generator>SettingsSingleFileGenerator</Generator>
+      <LastGenOutput>Settings.Designer.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="SessionRecords\CloudEndpointTests\" />
```

Comment:
Why do we need include in csproj?

Reply:
I believe these were automatically added when running the initial test recording, but I removed them in the latest commit and it should be resolved now.